src/github.com/stellar/horizon/resource/account.go

```diff
@@ -71,3 +72,16 @@ func (this *Account) Populate(
 	this.Links.Data.PopulateTemplated()
 	return
 }
+
+// GetData returns decoded value for a given key. If the key does
+// not exist, empty slice will be returned.
+func (this *Account) GetData(key string) []byte {
+	data, exists := this.Data[key]
+	if !exists {
+		return nil
+	}
+
+	// We assume that horizon always returns valid base64 encoded string
+	decoded, _ := base64.StdEncoding.DecodeString(data)
```

Comment:
Unfortunately, putting validation in the ingester won't help.  Someone may go in and overwrite a value with SQL.

Granted, we cannot protect against most tampering that may occur in the DB, but where we can, we should.  

---

Why can't you handle the  errors?  Every assumption you make about outside state is just another place for your code to break.

Why do you need a method to get a decoded value without having to handle errors?  The use case isn't clear to me... what function are you passing the method to?  I'm, resistant because anytime I pretend an error can't occur I end up getting bitten in the ass with a hard to track down error.

I would prefer instead we provide a `MustGetData` variant that triggers a panic for cases where you can't abide a corrupted value.  It fail fasts and is idiomatic.